SU-SWS/jumpstart_ui
# Update to Decanter 6.0.4

# READY FOR REVIEW 

# Summary
- Pull in the latest styles and accessibility fixes from Decanter

# Needed By (Date)
- Thursday

# Urgency
- Med/High

# Steps to Test

1. Review code and call out anything that changed that might be off
2. Check out this branch and clear all caches
3. Visually review the following elements
* Brand Bar
* Hero Banner
* Local Footer

# Affected Projects or Products
- D8CORE

# Associated Issues and/or People
- D8CORE-1198 & D8CORE-1199
- @cjwest 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
